enterprise-uda-installer-archives-mapping.ttl

```diff
@@ -11736,7 +11736,7 @@ source: a schema:CreativeWork ;
     a oplinst:FileSizeSpecification, schema:CreativeWork ;
     schema:name "File Size Details -- mxj3zzzz.dmg on Mac OS X 10.7.x (Universal) (with location Amazon S3 HTTP Depot)" ;
     oplinst:hasMeasurementUnit "Bytes" ;
-    oplinst:hasMeasurementValue "13231233"^^xsd:integer ;
+    oplinst:hasMeasurementValue "2307457"^^xsd:integer ;
```

Comment:
The new archive is 11 MB smaller than previous?  That suggests to me that something's gone wrong, or inadvertently been left out, when building this archive.

Reply:
The size is correct as the previous (current live) installers was actually includes the JDBC-ODBC Lite Bridge Shared library components which should not have been included, so now they are removed such that it just includes the JAR file require for MT JDBC client connection ...